classes/autoptimizeStyles.php

```diff
@@ -1210,6 +1217,14 @@ public function minify_single( $filepath, $cache_miss = false )
             // Now minify...
             $cssmin   = new autoptimizeCSSmin();
             $contents = trim( $cssmin->run( $contents ) );
+            
+            // Check if minified cache content is empty. //
+	        if ( empty( $contents ) ) {
+                // Do not store empty cache and remove url from cache list. //
+		        $url = '';
```

Comment:
you can simply do `return '';` here, no? :)

Reply:
Yes, fine by me! PHPStorm was also suggesting this as a improvement.